cdn/snippets.py

```diff
@@ -23,199 +23,195 @@
 
 import argparse
 import base64
-import datetime
+from datetime import datetime
 import hashlib
 import hmac
-
-from six.moves import urllib
+from urllib.parse import parse_qs, urlsplit
 
 
 # [START cloudcdn_sign_url]
-def sign_url(url, key_name, base64_key, expiration_time):
+def sign_url(
+    url: str, key_name: str, base64_key: str, expiration_time: datetime
```

Comment:
For readability please split into multiple lines ([see here for an example](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/0ecdffa9b444aa01fab18324709cec9095c4bef1/compute/client_library/snippets/instances/create.py#L49-L55))